TODO.md

```diff
@@ -1 +1,2 @@
-See [issues](https://github.com/uniqush/uniqush-push/issues)
+Use NSQ to send push messages
```

Comment:
Any reason why we need to change TODO? Right now, we are tracking our progress in the issues.

Why we want to NSQ?